deps/media-playback/media-playback/media.c

```diff
@@ -749,17 +960,22 @@ static void *mp_media_thread_start(void *opaque)
 {
 	mp_media_t *m = opaque;
 
-	if (!mp_media_thread(m)) {
-		if (m->stop_cb) {
+	if (!mp_media_thread(m))
+		if (m->stop_cb) 
 			m->stop_cb(m->opaque);
-		}
-	}
 
 	return NULL;
 }
 
-static inline bool mp_media_init_internal(mp_media_t *m,
```

Comment:
Why did you remove the `inline` from `mp_media_init_internal`?

Reply:
I couldn't see why I did either, so I reverted the change. Tested and don't see any issues.